Philips.CodeAnalysis.Common/TestHelper.cs

```diff
@@ -71,5 +74,19 @@ public bool IsTestMethod(AttributeListSyntax attributes, SyntaxNodeAnalysisConte
 			}
 			return hasAttribute;
 		}
+
+		public static bool IsTestProject(Compilation compilation)
+		{
+			// Check if this is a test project by looking for common test framework references
+			var referencedAssemblyNames = new HashSet<string>(
+				compilation.ReferencedAssemblyNames.Select(name => name.Name),
+				StringComparer.OrdinalIgnoreCase);
+
+			return referencedAssemblyNames.Contains("Microsoft.VisualStudio.TestPlatform.TestFramework") ||
+				   referencedAssemblyNames.Contains("MSTest.TestFramework") ||
+				   referencedAssemblyNames.Contains("NUnit.Framework") ||
+				   referencedAssemblyNames.Contains("xunit") ||
```

Comment:
As a sanity check, are these strings ("Nunit.Framework", "xunit", etc.) already defined somewhere in this project?